generator-mediator-js/app/templates/mediatorTemplate/config/_mediator.json

```diff
@@ -0,0 +1,67 @@
+{
+  "urn": "<%= mediatorUUID %>",
+  "version": "0.0.1",
+  "name": "<%= mediatorName %>",
+  "description": "<%= mediatorDesc %>",
+  "defaultChannelConfig": [
+    {
+      "name": "<%= mediatorName %>",
+      "urlPattern": "<%= defaultChannelPath %>",
+      "alerts": [],
+      "txRerunAcl": [],
+      "txViewFullAcl": [],
+      "txViewAcl": [],
+      "properties": [],
+      "matchContentTypes": [],
+      "routes": [
+        {
+          "name": "<%= mediatorName %> Route",
+          "host": "<%= mediatorRouteHost %>",
+          "path": "<%= mediatorRoutePath %>",
+          "port": "<%= configPort %>",
+          "primary": true,
+          "type": "http"
+        }
+      ],
+      "allow": ["admin"],
+      "type": "http"
+    }
+  ],
+  "endpoints": [
+    {
+      "name": "<%= mediatorName %> Route",
+      "host": "<%= mediatorRouteHost %>",
+      "path": "<%= mediatorRoutePath %>",
+      "port": "<%= configPort %>",
+      "primary": true,
+      "type": "http"
+    }
+  ],
+  "configDefs": [
+    {
+      "param": "sampleApplication",
+      "displayName": "sampleApplication Server",
+      "description": "The sampleApplication server with purpose sample",
+      "type": "struct",
+      "template": [
+        {
+          "param": "url",
+          "displayName": "URL",
+          "description": "The base URL (e.g. https://sampleApplication.io)",
+          "type": "string"
+        }, {
+          "param": "routePath",
+          "displayName": "routePath",
+          "description": "The mediator channel route path",
+          "type": "string"
+        }
+      ]
+    }
+  ],
+  "config": {
+    "sampleApplication": {
+      "url": "http://localhost:7001",
+      "path": "/routePath"
+    }
```

Comment:
A common thing for a mediator to do is to forward requests to an upstream server after changing something in the request. Maybe we should include an `upstreamUrl` config option rather than this sampleApplication struct.